packages/core/src/stepper/Stepper.tsx

```diff
@@ -0,0 +1,50 @@
+import { useComponentCssInjection } from "@salt-ds/styles";
+import { useWindow } from "@salt-ds/window";
+import { clsx } from "clsx";
+import { type ComponentProps, forwardRef, useContext } from "react";
+import { makePrefixer } from "../utils";
+import StepperCSS from "./Stepper.css";
+import {
+  StepperOrientationContext,
+  StepperProvider,
+} from "./internal/StepperProvider";
+
+const withBaseName = makePrefixer("saltStepper");
+
+export type StepperOrientation = "horizontal" | "vertical";
+
+export interface StepperProps extends ComponentProps<"ol"> {
```

Comment:
I don't know whether there is a preference around `ComponentPropsWithoutRef` vs this